main.go

```diff
@@ -161,7 +153,7 @@ func run() error {
 	prowClient := dynamicClient.Resource(schema.GroupVersionResource{Group: "prow.k8s.io", Version: "v1", Resource: "prowjobs"})
 
 	// Config and Client to access release images
-	releaseConfig, err := loadKubeconfigFromFlagOrDefault(opt.ReleaseClusterKubeconfig, prowJobKubeconfig)
+	releaseConfig, err := manager.LoadKubeconfigFromFlagOrDefault(opt.ReleaseClusterKubeconfig, prowJobKubeconfig)
```

Comment:
Same as above.  Loading kubeconfigs shouldnt be part of the Manager.

Reply:
Moved the method to `main.go`